### PR TITLE
Remove tram from Research Hallway

### PR DESCRIPTION
**Pull request by tigercat2000**
MT/TS added one of it's trams to the research hallway, mostly as a
showcase, and the actual usefulness:annoyingness ratio was not taken into
account at all, mostly due to rose-tinted glasses after spending a long
time coding the modular connector system. Multiple people have pointed out
that it was poorly thought out, and as I look at it in the map editor, I
agree- it is a bit of an eyesore, serves no functional purpose, and breaks
whenever someone gets in it's way, which is common.

MT/TS will most likely return to the map some time in the future, when I
find a functionally useful place for it- Having it in research did
accomplish one of my goals of subjecting it to actual playtesting,
although not very much data has been gathered.

TL:DR; Aahh this actually wasn't a good idea and is hurting my & other's eyes, removed.

### CODE REVIEW

**Comment by fj45** [comment]
Mining could possibly use one.

Big thing is some good sprites for it.
